Docker/run.sh

```diff
@@ -3,16 +3,20 @@
 # Trap these SIGNALs and run teardown
 trap 'trap_handler $@' INT TERM
 
+event_dumped=""
 dump_event() {
-  if [ -r /tmp/sd_event.json ]; then
+  if [ -r /tmp/sd_event.json ] && [ -z "$event_dumped" ]; then
```

Comment:
Question, does `event_dumped` set elsewhere? Otherwise, how does this ever get called?

Reply:
it's set up above

Reply:
Nevermind, it means `dump_event` only gets called 1 time